### PR TITLE
fix: allow to filter with empty or undefined value

### PR DESCRIPTION
**Pull request by alice-sevin**
Removed minLength on filter input to allow empty or undefined value.
Empty input won't raise error anymore.

### CODE REVIEW

File: tests/unit/filter-step-form.spec.ts [hunk 1]
```diff
@@ -20,12 +20,12 @@ describe('Filter Step Form', () => {
       data: {
         editedStep: {
           name: 'filter',
-          condition: { column: 'toto', operator: 'eq', value: '' },
+          condition: { column: 'toto', operator: 'eq', value: {} },
```
Comment (davinov) [inline tests/unit/filter-step-form.spec.ts:23]: OK! IMO, we could have a more relevant test on this validation.
With the widgets available in the forms, I don't see any case where users can send an object as value (`value: {}`)
If you want to keep a validation test, I suggest testing the column name instead (it shouldn't be empty) https://github.com/ToucanToco/weaverbird/pull/569/files#diff-41a18ffe8fcf0208fad725aaee286fd9R91-R93

**Comment by alice-sevin** [comment]
[As an AppBuilder, I can use templating even when my variable is undefined](https://trello.com/c/Lei0eyD1/4240-as-an-appbuilder-i-can-use-templating-even-when-my-variable-is-undefined)

File: tests/unit/filter-step-form.spec.ts [hunk 2]
```diff
@@ -153,6 +153,35 @@ describe('Filter Step Form', () => {
...
+      formSaved: [
+        [
+          {
+            name: 'filter',
+            condition: { column: 'columnA', operator: 'eq', value: '' },
+          },
+        ],
+      ],
+    });
+  });
```
Comment (davinov) [inline tests/unit/filter-step-form.spec.ts:183]: :+1: Thanks for this test :100:

File: tests/unit/filter-step-form.spec.ts (hunk 1 above)
Comment (alice-sevin) [reply tests/unit/filter-step-form.spec.ts:23]: Perfect, let's go for it!

**Pull request merged**